web/beacon-app/src/features/teams/types/member.ts

```diff
@@ -12,3 +12,10 @@ export type Member = {
 
 export type MemberRole = keyof typeof MEMBER_ROLE;
 export type MemberStatus = keyof typeof MEMBER_STATUS;
+
+export enum MemberStatusEnum {
+  ACTIVE = 'Active',
```

Comment:
Currently in the backend, there are two member status options and they are `Confirmed` and `Pending`.